policies/bundles/DocumentCSFAContentValid/metric.rego

```diff
@@ -0,0 +1,19 @@
+package clouditor.metrics.document_csfa_content_valid
+
+import rego.v1
+import input as document
+
+default applicable := false
+
+default compliant := false
+
+applicable if {
+	# the resource type should be a VM
+	"Document" in document.type
```

Comment:
Since the metric is expicitly called CSAFXXX I would here also check the two advisory documents explicitly and not ever document in general.